File: Python/ExcelSheetColumnTitleTest.py
from unittest import TestCase

from ExcelSheetColumnTitle import ExcelSheetColumnTitle


class TestExcelSheetColumnTitle(TestCase):
    def test_convertToTitle(self):
        e = ExcelSheetColumnTitle()
        
        self.assertEqual(e.convertToTitle(1), "A")

        self.assertEqual(e.convertToTitle(2), "B")

        self.assertEqual(e.convertToTitle(26), "Z")

        self.assertEqual(e.convertToTitle(27), "AA")

        self.assertEqual(e.convertToTitle(28), "AB")

File: Golang/ExcelSheetColumnTitle/ExcelSheetColumnTitle_test.go
package ExcelSheetColumnTitle

import "testing"

func TestConvertToTitle(t *testing.T) {

	if convertToTitle(1) == "A" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if convertToTitle(2) == "B" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if convertToTitle(26) == "Z" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if convertToTitle(27) == "AA" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if convertToTitle(28) == "AB" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

}
